spec/index.bs

```diff
@@ -1400,7 +1400,7 @@ implicitly [$valid to use with|unusable$].
         1. [=set/Append=] {{GPUFeatureName/"texture-formats-tier1"}} to |features|
     1. If |features| contains {{GPUFeatureName/"texture-formats-tier1"}}:
         1. [=set/Append=] {{GPUFeatureName/"rg11b10ufloat-renderable"}} to |features|
-    1. If |adapter| is [=Core-defaulting=] or |features| contains {{GPUFeatureName/"core-features-and-limits"}}:
```

Comment:
This was intended to allow Compat-defaulting adapters to request core-features-and-limits. Is this still possible?

Reply:
This wasn't doing anything, it was saying:
> if `"core-features-and-limits"` is in `features`,
> append `"core-features-and-limits"` to `features`.

where `features` is the `GPUDeviceDescriptor.requiredFeatures` requested by the user.

Compat-defaulting adapters allow requesting `"core-features-and-limits"` because it's in the adapter feature list. The requestDevice validation:

> The set of values in descriptor.requiredFeatures must be a subset of those in adapter.[[features]].